Userland/Libraries/LibAudio/FlacWriter.cpp

```diff
@@ -124,6 +126,39 @@ ErrorOr<void> FlacWriter::set_metadata(Metadata const& metadata)
     return add_metadata_block(move(vorbis_block), 0);
 }
 
+void FlacWriter::sample_count_hint(size_t sample_count)
+{
+#define oom_warning "FLAC Warning: Couldn't use sample hint to reserve {} bytes padding; ignoring hint."
```

Comment:
Can we just use a `constexpr StringView oom_warning = "FLAC Warning: Couldn't use sample hint to reserve {} bytes padding; ignoring hint."sv` here? I don't see a reason to use a #define.